ipv6/node_test.go

```diff
@@ -1037,16 +1043,19 @@ func TestDeleteBothChildren(t *testing.T) {
 		48,
 	}
 	trie, err := trie.Insert(key, nil)
+	assert.Nil(t, err)
 	leftChild := Prefix{
 		_a("1720:16:200::"),
 		49,
 	}
 	trie, err = trie.Insert(leftChild, nil)
+	assert.Nil(t, err)
 	rightChild := Prefix{
-		_a("1720:16:200::8000"),
+		_a("1720:16:200:8000::"),
```

Comment:
`staticcheck` found this error indirectly. It complained that the value of err on L1051 went uncheck, so I checked it for nil. When it wasn't nil, I found this issue.